amf-shapes/shared/src/main/scala/amf/shapes/internal/annotations/Annotations.scala

```diff
@@ -15,6 +15,15 @@ object ParsedJSONSchema extends AnnotationGraphLoader {
     Some(ParsedJSONSchema(value))
 }
 
+case class IsAVROSchema(avroType: String) extends EternalSerializedAnnotation {
```

Comment:
I would change the name of this to "AVROSchemaType" or something more descriptive

Reply:
Done 👌🏽